Possible missing include for std::numeric_limits

In `Intermediate.cpp` there's a use of `std::numeric_limits` to get infinity, but there's no corresponding include of `<limits>` in that file or its headers as far as I can see. I believe normally this gets covered by the PCH which includes `<climits>`, though I'm embedding glslang in my project so it acts as if ENABLE_PCH=OFF.

https://github.com/KhronosGroup/glslang/blob/8dffe4c195918813d3aa4c557c2853e55e9ca037/glslang/MachineIndependent/Intermediate.cpp#L2439

which leads to an error when I compile:

```
/p/renderdoc/renderdoc/3rdparty/glslang/glslang/MachineIndependent/Intermediate.cpp:2431:22: error: no member named 'numeric_limits' in namespace 'std'
 2431 |             d = std::numeric_limits<double>::infinity();
      |                 ~~~~~^
```

That said, I tried to reproduce this upstream and I could not. It looks like some property of the build environment in glslang by default ends up with `<limits>` included, where it isn't in my embedded copy even with the same compiler version. It might be due to C++ version being compiled with (I have added local downstream hacks to compile on an older C++ version/compilers).

It's totally fine if you don't think this is worth fixing, I have fixed it locally either way.

--- a/glslang/MachineIndependent/Intermediate.cpp
+++ b/glslang/MachineIndependent/Intermediate.cpp
@@ -46,6 +46,7 @@
 #include "propagateNoContraction.h"
 
 #include <cfloat>
+#include <limits>
 #include <utility>
 #include <tuple>
 

